StortSpelProjekt/Engine/src/Renderer/DX12Tasks/ForwardRenderTask.cpp

```diff
@@ -54,21 +60,35 @@ void FowardRenderTask::Execute()
 	DescriptorHeap* renderTargetHeap = m_DescriptorHeaps[DESCRIPTOR_HEAP_TYPE::RTV];
 	DescriptorHeap* depthBufferHeap  = m_DescriptorHeaps[DESCRIPTOR_HEAP_TYPE::DSV];
 
-	D3D12_CPU_DESCRIPTOR_HANDLE cdh = renderTargetHeap->GetCPUHeapAt(m_BackBufferIndex);
+	// RenderTargets
+	const unsigned int SwapChainIndex = swapChainRenderTarget->GetDescriptorHeapIndex();
```

Comment:
Capital letter on this variable, is this intended? Else change this for consistency.